lib/parallel_tests/cucumber/failures_logger.rb

```diff
@@ -7,14 +7,16 @@ class FailuresLogger < ::Cucumber::Formatter::Rerun
       include ParallelTests::Gherkin::Io
 
       def initialize(runtime, path_or_io, options)
+        super
         @io = prepare_io(path_or_io)
       end
 
-      def after_feature(feature)
-        unless @lines.empty?
-          lock_output do
-            @lines.each do |line|
-              @io.puts "#{feature.file}:#{line}"
+      def done
+        return if @failures.empty?
+        lock_output do
+          @failures.each do |file, lines|
+            lines.each do |line|
+              @io.print "#{file}:#{line} "
```

Comment:
Noap, cucumber does not support this anymore, probably they "broke" it somehow...
So for example if we have in the file something like:
feature1:10
feature2:10
then only feature1:10 will be run.
Instead we have them without \n, then works ok.

Also, this has been "tried" to be fixed in the PR: https://github.com/grosser/parallel_tests/pull/398